api-docs/rst/dev-guide/concepts.rst

```diff
@@ -217,6 +254,51 @@ servers.
  .. _List Load Balancer Details: http://docs.rackspace.com/loadbalancers/api/v1.0/clb-getting-started/content/List_LB_Details.html
 
 
+.. _stack-parameters:
+
+Stack parameters
+^^^^^^^^^^^^^^^^
+
+Stack parameters specify the configuration of the stacks that will be
+created during the scaling process. Note the ``stack`` arguments are
+directly passed to Cloud Orchestration when creating a stack. For more
```

Comment:
stacks to be created during the scaling process. Note that the ``stack`` arguments are passed directly to Cloud Orchestration during stack creation.